drltrace/CMakeLists.txt

```diff
@@ -109,3 +109,12 @@ endif ()
 install(TARGETS drltrace DESTINATION "${INSTALL_BIN}"
   PERMISSIONS ${owner_access} OWNER_EXECUTE GROUP_READ GROUP_EXECUTE
   WORLD_READ WORLD_EXECUTE)
+
+##################################################
+# drltrace test
+
+if (BUILD_TOOL_TESTS)
+  get_target_property(app_path drsyscall_app LOCATION${location_suffix})
+  get_target_property(drltrace_path drltrace LOCATION${location_suffix})
+  add_test(drltrace ${drltrace_path} -- ${app_path})
```

Comment:
Is that an internal error at AppVeyor or not?

Build started
git clone -q --depth=10 https://github.com/DynamoRIO/drmemory.git C:\projects\drmemory
git fetch -q origin +refs/pull/1964/merge:
fatal: unable to access 'https://github.com/DynamoRIO/drmemory.git/': Unknown SSL protocol error in connection to github.com:443 
Command exited with code 128